pkg/config/config.go

```diff
@@ -410,9 +412,14 @@ func initConfig(config Config) {
 	config.SetKnown("apm_config.receiver_port")
 	config.SetKnown("apm_config.env")
 	config.SetKnown("apm_config.apm_non_local_traffic")
+	config.SetKnown("apm_config.apm_dd_url")
 	config.SetKnown("apm_config.extra_sample_rate")
 	config.SetKnown("apm_config.ignore_resources")
+	config.SetKnown("apm_config.log_file")
```

Comment:
As discussed, let's revert all the changes to this file and we will add them as part of a separate PR.

Reply:
Done